snapshot/snapshots/fake_checkouter.go

```diff
@@ -7,6 +7,7 @@ import (
 	"github.com/scootdev/scoot/snapshot"
 )
 
+// Create a Checkouter that always fails due to invalid path
```

Comment:
It doesn't actually fail. I'm not sure why we say it's a noop struct and then say MakeInvalid...